app/views/users/nbp_wallet/connect.html.slim

```diff
@@ -0,0 +1,24 @@
+.header.mb-5
+  .std-heading
+    i.fa-solid.fa-wallet style='color: #444444'
```

Comment:
I think we are currently moving away from inline styles (rather using CSS classes). See #1359 as well.

_This comment applies to multiple occasions in this file._

Reply:
Done (mostly, there is a style=width:100% left but it seemed unnecessary to make a CSS class for that)

Reply:
You can use a Bootstrap class for that: `w-100`. It will just work and only applies `width: 100%` 🙂